apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -128,8 +144,15 @@ export function LoginForm({ email, token }: Props) {
           </Text>
         </Link>
 
-        <Button mt={60} inherit loading={isLoading} submit data-test-id="submit-btn">
-          Sign In
+        <Button
+          submit
+          mt={60}
+          inherit
+          loading={isLoading || isLoadingAcceptInvite}
+          disabled={isLoadingAcceptInvite}
+          data-test-id="submit-btn"
+        >
+          {invitationToken ? 'Sign In & Accept' : 'Sign In'}
```

Comment:
Would it make more sense to have the text `Accept & Sign In`? In that order.

Reply:
I did it the same way like it is in the sign up form